### PR TITLE
Added bool query.

### LINKED ISSUE #10: Implement Boolean Query
https://www.elastic.co/guide/en/elasticsearch/reference/1.6/query-dsl-bool-query.html

### PR DESCRIPTION
**Pull request by TomChaton**
Resolves #10

### CODE REVIEW

**Comment by TomChaton** [comment]
Not sure why coveralls failed. I made the coverage checks a lot stricter, but this PR increases coverage. Hmmm

**Comment by TomChaton** [comment]
Derp. I increased the coverage threshold to 99% instead of 95%

File: src/Lib/Collection.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?php
```
Comment (luispabon) [inline src/Lib/Collection.php:1]: Both Collection and DocumentCollection are also on 11 :confused:

Comment (luispabon) [inline src/Lib/Collection.php:1]: Ah crap I noticed now it does include this PR.No problemo

**Comment by luispabon** [comment]
Fix that typo on ShouldSucceedsTest and I'll merge the lot :+1:

**Comment by TomChaton** [comment]
Cheers @luispabon. You want me to cherry-pick all these PRs into one branch to save us from rebase/merge hell?

**Comment by luispabon** [comment]
For as long as there was no squashing involved they should all be clean
merges, shouldn't they?
On 10 Aug 2015 8:48 am, "Tom Croft" notifications@github.com wrote:

> Cheers @luispabon https://github.com/luispabon. You want me to
> cherry-pick all these PRS into one branch to save us from rebase/merge
> hell?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tomcroft/tantrum_elastic/pull/78#issuecomment-129343055
> .

## 

1 Neal Street London WC2H 9QL

www.rightster.com | @Rightster https://twitter.com/rightster | LinkedIn 
https://www.linkedin.com/company/rightster

Rightster Limited (incorporated in England with company number 07634543) 
has its registered office at 1 Neal Street London WC2H 9QL. Rightster 
Limited is a subsidiary of Rightster Group plc (incorporated in England and 
Wales with company number 08754680).

**Pull request merged**